File: src/main/java/com/apiApp/CateringFacilityAPI/model/jpa/ApiInvoice.java
package com.apiApp.CateringFacilityAPI.model.jpa;

import javax.persistence.*;
import javax.validation.constraints.NotNull;
import java.time.LocalDateTime;

@Entity
@Table(name = "api_invoices")
public class ApiInvoice extends Invoice {

    @NotNull
    @ManyToOne
    private Developer developer;

    public Developer getDeveloper() {
        return developer;
    }

    public void setDeveloper(Developer developer) {
        this.developer = developer;
    }


}

File: frontend/src/app/model/ApiInvoice.ts
import {Developer} from "./Developer";
import {SubscriptionPackage} from "./SubscriptionPackage";
import {Invoice} from "./Invoice";

export class ApiInvoice extends Invoice{
  public developer: Developer;
  getCreatedAt(){
    // this.createdAt;
    return 'Bojan';
  }
}
